src/sys/stat.rs

```diff
@@ -209,7 +209,8 @@ pub fn utimes<P: ?Sized + NixPath>(path: &P, atime: &TimeVal, mtime: &TimeVal) -
 /// # References
 ///
 /// [lutimes(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/lutimes.html).
-#[cfg(not(target_os = "android"))]
+#[cfg(not(any(target_os = "android",
```

Comment:
As far as I understand `lutimes` doesn't exist on OpenBSD. For one thing I cannot find any reference of it in the entire OpenBSD code-base (easy to search on https://github.com/openbsd/src) and previous discussions here: https://github.com/rust-lang/libc/pull/790 and here: https://github.com/alexcrichton/filetime/issues/33 led to the same conclusion.

I can change to the whitelist version of the `cfg` attribute, no problem on my side, but I don't think it should include `android` since before my change it was `not(target_os = "android")`.

Reply:
Yes, definitely leave Android out, I just copy/pasted from somewhere else and forgot to remove it; good catch.

Reply:
Changed to whitelist in 25fa7db .